Appraisals

```diff
@@ -1514,6 +1523,7 @@ elsif ruby_version?('3.2')
     gem 'typhoeus'
     gem 'que', '>= 1.0.0'
     gem 'net-smtp'
+    gem 'nokogiri', platform: :ruby # binary gem has max ruby version constraint excluding previews
```

Comment:
Minor: Maybe leave here a `TODO: Remove once Ruby 3.2 final is out`?

Reply:
Done

Reply:
Thank you!